radio/src/gui/colorlcd/radio_setup.cpp

```diff
@@ -376,14 +376,15 @@ void RadioSetupPage::build(FormWindow * window)
     grid.nextLine();
 #endif
   
+#if defined(INTERNAL_GPS)
```

Comment:
I think this could be made conditional on external serial port config. I‘m trying to avoid compilation settings as possible, so our generic build are as polyvalent as possible.

Reply:
Oh wait a sec! Would this apply to an external GPS as well? (like those connected to an external serial port?)

Reply:
I cannot speak for other radios, but for TX16S, IMHO there is no difference, as the GPS must be connected to UART6. If one uses the internal pads (e.g. from Bluetooth) or hooks it up via AUX2, the signal path is the same.
Also, presently cannot use simultaneously Bluetooth *AND* TxGPS due to this limitation (at least again on TX16S).

It might very well be a different story on X10/X12 ?

Reply:
> I think this could be made conditional on external serial port config. I‘m trying to avoid compilation settings as possible, so our generic build are as polyvalent as possible.

I frankly have no idea what I should make with this ... it seems so off to me that I wonder what the aim is of repeating it over and over again ... 
is it just a comment for having said it or do you somehow exepect some action?

Reply:
No, I really think we should do it. But well, if you don't feel like it, or do not know how, I'll do it later.

Reply:
you think as part of this PR?

Reply:
I agree that having the GPS stuff in the menu on transmitters without GPS is annoying. I think the point being made by  Raphael is that it would be better to not lock it behind a compile time define, but behind an option on the hardware page. On the TX16S, I don't know how you would enable the GPS (I've not tried, not looked at how you would so it, but there is certainly no mention of on it in either of the Serial or Serial 2 lists.

Reply:
the point he made is clear. It just doesn't make any sense IMHO to argue it here.

Reply:
You don't think there's any point suggesting that you don't use a compile time conditional (as your PR specifically does) and instead that it could be configured by the GUI elsewhere? What am I missing here?

Reply:
I don't dispute the general idea, but that it doesn't make any sense to argue it for this PR...

Reply:
Ok ok, let's do it later!